### DISCUSSION TITLE
How to create alarm rules about service instance downtime, help

### DISCUSSION THREAD

**Discussion by DemoCap** [apache/skywalking#8070] [Q&A] [answered]
I want to create an alarm rule for service instance downtime. I configured it like this, but it didn't take effect

instance_jvm_cpu_rule:
    metrics-name: instance_jvm_cpu
    threshold: 0 #阈值
    op: "==" #操作符
    period: 5 #单位告警间隔(单位：分钟)，即在该单位分钟内合计触发count次满足条件
    count: 1 #合计多少次会触发告警
    silence-period: 5
    #only-as-condition: true #指定是否发送告警
    message: 服务实例{name}在过去一分钟内无cpu活动，疑似宕机

**Comment by wu-sheng** [comment, marked as answer]
I never tried to set alarm rule for CPU. This is a strange case.
It is better to debug locally first.

**Reply by DemoCap** [reply]
I passed the local test and the alarm rule is invalid (even if it is in the oal rule). In fact, I just want to monitor the service downtime and trigger the alarm. Is there any other solution? Please give me some suggestions

**Reply by wu-sheng** [reply]
Ideally, I want to recommend using event and health alarm, rather than CPU usage.

https://skywalking.apache.org/docs/main/v8.8.1/en/concepts-and-designs/event/

**Reply by DemoCap** [reply]
thanks.
